### PR TITLE
llm-proxy: init some ops docs

### PR DESCRIPTION
**Pull request by bobheadxi**
First step in documenting https://github.com/sourcegraph/sourcegraph/issues/50726. Will follow up with more as we finalize more details.

### CODE REVIEW

File: content/departments/engineering/teams/cody/index.md [hunk 1]
```diff
@@ -3,3 +3,7 @@
 Placeholder page for the cody team.
 
 - [About Cody FAQ](about-cody-faq.md)
+
+Managed Cody services:
+
+- [LLM-proxy](llm-proxy/index.md) (completions.sourcegraph.com and completions.sgdev.org)
```
Comment (eseliger) [inline content/departments/engineering/teams/cody/index.md]: Love that this is a list :comfy:

File: content/departments/engineering/teams/cody/llm-proxy/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# LLM-proxy
+
+> WARNING: This project is currently a work in progress - reach out to #wg-llm-proxy for more information.
+
+LLM-proxy is a managed proxy service that routes requests to various LLM backends such as Anthropic and OpenAI.
```
Comment (eseliger) [inline content/departments/engineering/teams/cody/llm-proxy/index.md]: openai not yet, should we say that yet?

**Review by eseliger — APPROVED** [review]
TY!

File: content/departments/engineering/teams/cody/index.md (hunk 1 above)
Comment (unknwon) [reply content/departments/engineering/teams/cody/index.md]: Is "LLM Proxy" or "LLM-proxy" or "LLM-Proxy"? Which one is the official one 🤔 personally I've thought it was "LLM Proxy".

Comment (eseliger) [reply content/departments/engineering/teams/cody/index.md]: LLM Proxy sgtm 😬

Comment (bobheadxi) [reply content/departments/engineering/teams/cody/index.md]: I don't mind, can change it to without the dash!

**Review by unknwon — APPROVED** [review]

**Pull request merged**